test/SILOptimizer/copyforward_ossa.sil

```diff
@@ -0,0 +1,845 @@
+  br bb2                                          // id: %9
+
+bb2:                                              // Preds: bb0 bb1
+  destroy_addr %0 : $*T                           // id: %10
+  %11 = tuple ()                                  // user: %12
+  return %11 : $()                                // id: %12
+}
+
+enum A<T> {
+  case Val(T)
+  init(_ val: T)
+}
+
+sil [transparent] @_TFO8enuminit1A3ValU__fMGS0_Q__FQ_GS0_Q__ : $@convention(thin) <T> (@in T, @thin A<T>.Type) -> @out A<T>
+
+//CHECK-LABEL: enuminit
+//CHECK-NOT: copy_addr
+//CHECK: return
+sil [ossa] @enuminit : $@convention(thin) <T> (@in T, @thin A<T>.Type) -> @out A<T> {
+bb0(%0 : $*A<T>, %1 : $*T, %2 : $@thin A<T>.Type):
+  %3 = alloc_stack $A<T>, var, name "sf"             // users: %10, %14, %16
+  // function_ref enuminit.A.Val <A>(enuminit.A<A>.Type)(A) -> enuminit.A<A>
+  %4 = function_ref @_TFO8enuminit1A3ValU__fMGS0_Q__FQ_GS0_Q__ : $@convention(thin) <τ_0_0> (@in τ_0_0, @thin A<τ_0_0>.Type) -> @out A<τ_0_0> // user: %9
+  %5 = metatype $@thin A<T>.Type                  // user: %9
+  %6 = alloc_stack $T                             // users: %7, %9, %12
+  copy_addr %1 to [initialization] %6 : $*T     // id: %7
+  %8 = alloc_stack $A<T>                          // users: %9, %10, %11
+  %9 = apply %4<T>(%8, %6, %5) : $@convention(thin) <τ_0_0> (@in τ_0_0, @thin A<τ_0_0>.Type) -> @out A<τ_0_0>
+  copy_addr [take] %8 to [initialization] %3 : $*A<T> // id: %10
+  dealloc_stack %8 : $*A<T>      // id: %11
+  dealloc_stack %6 : $*T         // id: %12
+  destroy_addr %1 : $*T                           // id: %13
+  copy_addr [take] %3 to [initialization] %0 : $*A<T> // id: %14
+  %15 = tuple ()                                  // user: %17
+  dealloc_stack %3 : $*A<T>      // id: %16
+  return %15 : $()                                // id: %17
+}
+
+//CHECK-LABEL: make_addronly
+//CHECK-NOT: copy_addr
+//CHECK: return
+sil hidden [ossa] @make_addronly : $@convention(thin) <T> () -> @out T {
+bb0(%0 : $*T):
+  %1 = alloc_stack $T, let, name "t"                   // users: %3, %4, %5
+  %2 = function_ref @f_out : $@convention(thin) <τ_0_0> () -> @out τ_0_0 // user: %3
+  %3 = apply %2<T>(%1) : $@convention(thin) <τ_0_0> () -> @out τ_0_0
+  copy_addr [take] %1 to [initialization] %0 : $*T // id: %4
+  dealloc_stack %1 : $*T         // id: %5
+  %6 = tuple ()                                   // user: %7
+  return %6 : $()                                 // id: %7
+}
+
+sil [ossa] @_TFSq4someU__fMGSqQ__FQ_GSqQ__ : $@convention(thin) <τ_0_0> (@in τ_0_0, @thin Optional<τ_0_0>.Type) -> @out Optional<τ_0_0>
+
+sil [ossa] @_TFsoi2neU__FTGSqQ__Vs26_OptionalNilComparisonType_Sb : $@convention(thin) <τ_0_0> (@in Optional<τ_0_0>, _OptionalNilComparisonType) -> Bool
+
+//CHECK-LABEL: option_init
+//CHECK: alloc_stack
+//CHECK: alloc_stack
+//CHECK: alloc_stack
+//CHECK: copy_addr
+//CHECK: copy_addr
+//CHECK-NOT: copy_addr
+//CHECK: alloc_stack
+//CHECK: alloc_stack
```

Comment:
Do you think you could try to do a replace-all of '//CHECK' -> '// CHECK'? I am pretty sure it will standardize the checks in the file without breaking anything.